### ISSUE TITLE
fix: StdoutException: Could not get terminal size, OS Error: Operation not supported on socket, errno = 102

### ISSUE THREAD

**Issue by rodrigopequeno** [VeryGoodOpenSource/very_good_cli#866]
**Description**
I'm using the `very_good test --coverage` command in a pipeline in Azure Devops, and it's returning this error when trying to write the message in the terminal:

"Thank you for using Very Good Ventures open source tools!
Don't forget to fill out this form to get information on future updates and releases here:
https://verygood.ventures/open-source/cli/subscribe-latest-tool-updates"

**Steps To Reproduce**

1. Install lastest version 1.17.0
2. Run `very_good test --coverage`

**Expected Behavior**

**Screenshots**
![image](https://github.com/VeryGoodOpenSource/very_good_cli/assets/23217960/614308a2-d473-4141-9773-2309777dfd7b)

**Additional Context**
Locally works normally

**Comment by alestiago** [comment]
Hi @rodrigopequeno ! Thanks for reporting the issue.

I would suggest adding an environment variable with a "CI" key and any value (I recommend CI=true), this will completely stop the message showing up.

If that doesn't work, a temporary option is to add a directory to your home named `.very_good_cli` with a file named `version` and the content of the file being the current version. This will avoid the message showing up since it is only displayed during the first update.

```sh
mkdir ~/.very_good_cli && echo -n '0.17.0' > ~/.very_good_cli/version
```

**Comment by rodrigopequeno** [comment]
Hi @alestiago, I chose to drowngrade the version for now. I will check these solutions as soon as possible.

**Comment by alestiago** [comment]
@rodrigopequeno this should now be fixed with Very Good CLI 0.17.1, feel free to reopen if the bug persists.
